CMakeLists.txt

```diff
@@ -560,19 +547,12 @@ if(QUDA_MPI OR QUDA_QMP)
       "Found MPIFORT/MPICC/MPICXX environment variables. If this is not what you want please use -DMPI_<LANG>_COMPILER and consult the cmake FindMPI documentation."
     )
   endif()
-  # we need to enable Fortran if we want to detect MPI_Fortran_COMPILER
-  if(QUDA_ARPACK OR QUDA_OPENBLAS)
-    enable_language(Fortran)
-  endif()
   find_package(MPI)
```

Comment:
Since this is guarded by `QUDA_MPI` should we make this `REQUIRED` ?  `QMP` should bring it transitively so perhaps we can soften this? But otherwise it is fine...